lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -71,8 +76,13 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 	portLayerClient = client.New(t, nil)
 	portLayerServerAddr = portLayerAddr
 
+	log.Info("Creating image store")
+	if err := createImageStore(); err == nil {
+		return err
+	}
+
 	// attempt to update the image cache at startup
-	log.Info("Refreshing image cache...")
+	log.Info("Refreshing image cache")
 	go func() {
```

Comment:
While we're here, this go routine could use some TLC.  First, lets do this synchronously and not in a go routine if it can be avoided.  Second, instead of a loop over retries, lets use[Ticker](https://golang.org/pkg/time/#Ticker).  How about something like https://play.golang.org/p/WsJ9EI8M7l.  Points if you can get rid of the `var` declaration.